churaverse-plugins-server/src/synchroBreakPlugin/message/synchroBreakResultMessage.ts

```diff
@@ -0,0 +1,20 @@
+import { IMainScene } from 'churaverse-engine-server'
+import { BaseMessage } from '@churaverse/network-plugin-server/message/baseMessage'
+import { SendableObject } from '@churaverse/network-plugin-server/types/sendable'
+
+export interface SynchroBreakResultData extends SendableObject {}
+
+/**
+ * ゲーム終了後の結果ウィンドウ表示メッセージ
+ */
+export class SynchroBreakResultMessage extends BaseMessage<IMainScene> {
+  public constructor(public readonly data: SynchroBreakResultData) {
+    super('synchroBreakResult', data)
+  }
+}
```

Comment:
送るデータがないのであれば、空のままメッセージを送った方がいいと思います

```suggestion
export class SynchroBreakResultMessage extends BaseMessage<IMainScene> {
  public constructor() {
    super('synchroBreakResult')
  }
}
```